webapp/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM debian:jessie
+ENV REFRESHED_AT 2015-05-18
+
+# update packages and prepare to build software
+RUN ["apt-get", "update"]
+RUN ["apt-get", "-y", "install", "build-essential"]
+
+ENV LANG en_US.utf8
+
+RUN ["apt-get", "-y", "install", "git"]
+
+# install latest python
+RUN ["apt-get", "-y", "install", "python3-dev", "python3-pip", "python3-pil", "python3-psycopg2"]
+
+# prepare postgreSQL support
+RUN ["apt-get", "-y", "install", "postgresql-server-dev-all"]
```

Comment:
yes, in an ideal world, it would, but psycopg2 is being compiled, which needs the headers. Will be fixed in the new setup as well.